packages/odyssey-react/src/components/Infobox/Infobox.module.scss

```diff
@@ -20,13 +20,13 @@
     ". actions";
   grid-template-columns: max-content 1fr;
   width: 100%;
-  max-width: calc(var(--FontLineLengthMax) + calc(var(--SpacePadding) * 2));
-  padding-block: var(--SpacePadding);
-  padding-inline: var(--SpacePadding);
+  max-width: calc(var(--FontLineLengthMax) + calc(var(--Padding) * 2));
+  padding-block: var(--Padding);
```

Comment:
When I did the initial conversion, I used the same variable for both since we expect them to be uniform in our theme. However, I think most other components have broken them up into `PaddingBlock` and `PaddingInline`. 

I'm not too opinionated since we haven't aligned on a standard.

Reply:
Fixed.